services/ui-src/src/page/action/RespondToRAI.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { ONEMAC_ROUTES, Workflow } from "cmscommonlib";
+
+const RAILink = {
+  [Workflow.ONEMAC_TYPE.MEDICAID_SPA]: ONEMAC_ROUTES.MEDICAID_SPA_RAI,
```

Comment:
I like that we define this here instead of needing TYPE_TO_RAI_ROUTE in common. Can we delete the definition from routes.js in common since it no longer appears to be needed there?